kolibri/plugins/management/assets/src/vue/index.vue

```diff
@@ -47,12 +48,17 @@
         }
         return null;
       },
-
+      isAdminOrSuperuser() {
+        if (this.kind[0] === UserKinds.SUPERUSER || this.kind[0] === UserKinds.ADMIN) {
+          return true;
+        }
+        return false;
+      },
     },
     vuex: {
       getters: {
         pageName: state => state.pageName,
-        isAdminOrSuperuser: state => state.core.is_admin_or_superuser,
```

Comment:
..... it shouldn't be used at all. i'm going to purge this from the code one sec